src/main/java/org/jenkinsci/plugins/liquibase/evaluator/DatabaseDocBuilder.java

```diff
@@ -66,7 +70,9 @@ public DatabaseDocGenerationCallback(Liquibase liquibase, Contexts contexts, Lab
 
         @Override
         public Void invoke(File f, VirtualChannel channel) throws IOException, InterruptedException {
-            f.mkdirs();
+            if (!f.mkdirs()) {
```

Comment:
`java.nio.Files` offers a better API which throws exceptions on errors. Here you basically supress the error, because the code will keep executing. `liquibase.generateDocumentation` will likely fail afterwards